data/hai/hai missions.txt

```diff
@@ -14,35 +14,107 @@ mission "First Contact: Hai"
+			choice
+				`	(Sure.)`
+				`	(Not right now.)`
+					defer
+			`	You walk up to a human merchant who is busy haggling with a Hai. They interrupt their conversation when they see you approaching, and the human gestures for you to join them. "First time down the rabbit hole?" he asks.`
+			`	You shake your head and explain how you met with the Unfettered that called themselves the 'True Hai'.`
+			`	The Hai says, "Ah, you have met our wayward sisters and brothers to the north. Is that what you have come to ask us about?`
+			choice
+				`	"Yes. Can you tell me why you are at war?"`
+				`	"No, I wanted to know more about your society. Is it as peaceful as it appears?"`
+					goto peaceful
+			`	He says, "I would not call it war. We launch no raids on them. We only defend when they attack. And, we supply them with food and other things needful for life. They are not our enemies, they are just those who do not fit in. You have surely seen what our society is like: safe, stable, predictable, deliberate. It suits most of us well. But there are those..."`
+				goto questions
+			label peaceful
+			`	"We are a people of peace, and you are free to visit us or even live among us. Our worlds provide enough bounty for all."`
+			`	You glance quickly at the human merchant to see his expression, wondering if these aliens can indeed be so benign, but his expression is calm and untroubled. "It's true," he says, "look around you. These buildings have stood for thousands of years. Hai society is so advanced, they need almost no resources to maintain it."`
+			choice
+				`	"Why do you allow humans to settle here?"`
+				`	"What do you ask for in return for letting humans settle here?"`
+			`	"Humans are a young species," the Hai says, "full of energy, full of new ideas. And the Hai are old, and everything we do is what we have done before. When humans go on vacation, they travel to a world with perfect weather, sunny every day. When Hai go on vacation, we visit the stormiest world, to find unpredictability and change. Humans are so strange, that to speak with a human is like a small vacation." It smiles, and you catch a glimpse of its massive incisors. Based on your knowledge of xenobiology you would guess that the Hai are herbivores, but you can't be certain.`
+			choice
+				`	"So I'm allowed to travel throughout your territory if I want?"`
+					goto north
+				`	"Can I ask more about the Unfettered now?"`
+					goto questions
+				`	"Thank you, but I have no more questions."`
+					decline
+			label questions
+			apply
+				set "Ask the Hai about the Unfettered: offered"
+			`	The Hai seems willing to explain more, what would you like to ask?`
```

Comment:
You just asked a question, so have the Hai actually respond to it instead of saying "they look like they'll give you an answer."
```suggestion
			`	"What exactly would you like to know?" the Hai responds.`
```